### ISSUE TITLE
Does use_gt_pelvis correspond to MPJPE relative to pelvis in the paper?

### ISSUE THREAD

**Issue by yihui-he** [karfly/learnable-triangulation-pytorch#30]
Does `use_gt_pelvis` correspond to MPJPE relative to pelvis in the paper? 
If so, why are the results worse with GT pelvis? 
Thanks!

**Comment by karfly** [comment]
Hi, @yihui-he. 
Not really. `use_gt_pelvis` means that volumetric cube will be placed in space according to ground-truth position of pelvis.

**Comment by yihui-he** [comment]
Thanks for the quick response.
I'm still confused about why relative to pelvis MPJPE results are worse than absolute MPJPE. For relative to pelvis MPJPE, the difference between the predicted pelvis and GT pelvis is set to 0, I guess.

**Comment by yurymalkov** [comment]
Hi @yihui-he , if you are talking about the results in the paper (Table 1 vs Table 2), the absolute positions have lower error because they are estimated independently of each other. 
For the case of the relative pelvis, the error of the pelvis estimation is accumulated with errors for each joint, leading to higher numbers.

Note that `use_gt_pelvis` is a train-time argument. As the evaluation is performed using triangulated pelvis position (not the `gt` ones), the results using the `use_gt_pelvis` might be worse due to train-test mismatch.

**Comment by yihui-he** [comment]
@yurymalkov , thank you! 
Yes, im talking about Table 1&2
So for the relative to pelvis MPJPE results, your multi-view methods estimate pelvis location + joints relative to pelvis. Then error is computed in world coordinate, am I right?

**Comment by yurymalkov** [comment]
@yihui-he 
The multi-view method always estimates the positions in world coordinates for all of the joints, including the pelvis. MPJPE relative to pelvis is the most common metric, so for a fair comparison in Table 1 we subtract the position of the pelvis from all joints.
We argue that for multi-view methods it makes more sense not to subtract the pelvis (as we did in Table 2).

**Comment by yihui-he** [comment]
Thanks for the explanation
